core/models.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+
+# Create your models here.
+class Book(models.Model):
+    book_name = models.CharField(max_length=255)
```

Comment:
pk can be added to make it more implicit. Refer here: https://peps.python.org/pep-0020/

Reply:
can also use uuid here